src/structural_pipeline/COMMIT-Filter/COMMIT_script.py

```diff
@@ -0,0 +1,82 @@
+# Script to run COMMIT and COMMIT2 on CATO output.
+# This script is based on the COMMIT wiki: https://github.com/daducci/COMMIT/wiki/COMMIT2
+
+import numpy as np
+import os
+import commit
+from commit import trk2dictionary
+import argparse
+
+argParser = argparse.ArgumentParser()
+
+argParser.add_argument("--dwiProcessedFile", help="preprocessed DWI file", required=True)
+argParser.add_argument("--dwiSchemeFile", help="DWI scheme file", required=True)
+argParser.add_argument("--fiberFile", help="Input fiber file", required=True)
+argParser.add_argument("--outputCommitDir", help="Output directory for COMMIT", required=True)
+argParser.add_argument("--intermediateConnectomeFile", help="Connectome file with number of streamline per connection (for internal use)", required=True)
+argParser.add_argument("--subjectDir", help="Subject directory", required=True)
+argParser.add_argument("--wmMaskFile", help="White matter mask file", required=True)
+argParser.add_argument("--regLambda", help="Regularisation parameter lambda", required=True)
+
+args = argParser.parse_args()
+
+# Import tractogram
+os.chdir(args.subjectDir)
+
+trk2dictionary.run(
+    filename_tractogram = args.fiberFile,
+    filename_mask = args.wmMaskFile,
+    path_out = args.outputCommitDir,
+    fiber_shift         = 0
```

Comment:
Shall we use [black](https://github.com/psf/black) and maybe [isort](https://pycqa.github.io/isort/) for any python files? That makes it easy to have uniform formatting across contributors. We can enforce it using github actions (see e.g. https://github.com/dutchconnectomelab/brain-reproducibility/tree/main/.github/workflows)